<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite id="0" name="C:\ReactWS\2019_dev_005\src\test\AppHeader.spec.js" errors="0" package="C:\ReactWS\2019_dev_005\src\test\AppHeader.spec.js" hostname="localhost" tests="1" failures="0" time="2.094" timestamp="2018-12-12T12:13:54">
    <properties>
    </properties>
    <testcase classname="Header Component" name="Renders Application Heading" time="0">
    </testcase>
    <system-out/>
    <system-err/>
  </testsuite>
  <testsuite id="1" name="C:\ReactWS\2019_dev_005\src\App.test.js" errors="0" package="C:\ReactWS\2019_dev_005\src\App.test.js" hostname="localhost" tests="1" failures="1" time="2.807" timestamp="2018-12-12T12:13:54">
    <properties>
    </properties>
    <testcase classname="renders without crashing" name="renders without crashing" time="0">
      <failure message="Invariant Violation: Could not find &quot;store&quot; in the context of &quot;Connect(App)&quot;. Either wrap the root component in a &lt;Provider&gt;, or pass a custom React context provider to &lt;Provider&gt; and the corresponding React context consumer to Connect(App) in connect options.
    at invariant (C:\ReactWS\2019_dev_005\node_modules\invariant\invariant.js:40:15)
    at Connect.renderWrappedComponent (C:\ReactWS\2019_dev_005\node_modules\react-redux\lib\components\connectAdvanced.js:162:32)
    at updateContextConsumer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:14944:19)
    at beginWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:15110:14)
    at performUnitOfWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17820:12)
    at workLoop (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17860:24)
    at renderRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17946:7)
    at performWorkOnRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18837:7)
    at performWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18749:7)
    at performSyncWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18723:3)
    at requestWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18592:5)
    at scheduleWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18401:5)
    at scheduleRootUpdate (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19069:3)
    at updateContainerAtExpirationTime (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19097:10)
    at updateContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19154:10)
    at ReactRoot.Object.&lt;anonymous&gt;.ReactRoot.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19416:3)
    at C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19556:14
    at unbatchedUpdates (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18952:10)
    at legacyRenderSubtreeIntoContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19552:5)
    at Object.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19613:12)
    at Object.render (C:\ReactWS\2019_dev_005\src\App.test.js:7:12)
    at Object.asyncJestTest (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\jasmine_async.js:108:37)
    at resolve (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)" type="AssertionError">
      </failure>
    </testcase>
    <system-out/>
    <system-err/>
  </testsuite>
  <testsuite id="2" name="C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js" errors="0" package="C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js" hostname="localhost" tests="2" failures="2" time="3.431" timestamp="2018-12-12T12:13:54">
    <properties>
    </properties>
    <testcase classname="PlayerForm Component" name="Number of input boxes" time="0">
      <failure message="Invariant Violation: Could not find &quot;store&quot; in the context of &quot;Connect(PlayerForm)&quot;. Either wrap the root component in a &lt;Provider&gt;, or pass a custom React context provider to &lt;Provider&gt; and the corresponding React context consumer to Connect(PlayerForm) in connect options.
    at invariant (C:\ReactWS\2019_dev_005\node_modules\invariant\invariant.js:40:15)
    at Connect.renderWrappedComponent (C:\ReactWS\2019_dev_005\node_modules\react-redux\lib\components\connectAdvanced.js:162:32)
    at updateContextConsumer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:14944:19)
    at beginWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:15110:14)
    at performUnitOfWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17820:12)
    at workLoop (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17860:24)
    at renderRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17946:7)
    at performWorkOnRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18837:7)
    at performWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18749:7)
    at performSyncWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18723:3)
    at requestWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18592:5)
    at scheduleWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18401:5)
    at scheduleRootUpdate (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19069:3)
    at updateContainerAtExpirationTime (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19097:10)
    at updateContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19154:10)
    at ReactRoot.Object.&lt;anonymous&gt;.ReactRoot.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19416:3)
    at C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19556:14
    at unbatchedUpdates (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18952:10)
    at legacyRenderSubtreeIntoContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19552:5)
    at Object.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19613:12)
    at Object.render (C:\ReactWS\2019_dev_005\node_modules\enzyme-adapter-react-16\build\ReactSixteenAdapter.js:352:114)
    at new ReactWrapper (C:\ReactWS\2019_dev_005\node_modules\enzyme\build\ReactWrapper.js:134:16)
    at Object.mount (C:\ReactWS\2019_dev_005\node_modules\enzyme\build\mount.js:21:10)
    at Object.mount (C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js:20:22)
    at Object.asyncJestLifecycle (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\jasmine_async.js:63:37)
    at resolve (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)" type="AssertionError">
      </failure>
      <failure message="TypeError: Cannot read property &apos;find&apos; of undefined
    at Object.find (C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js:26:20)
    at Object.asyncJestTest (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\jasmine_async.js:108:37)
    at resolve (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)" type="AssertionError">
      </failure>
    </testcase>
    <testcase classname="PlayerForm Component" name="Name input function call" time="0">
      <failure message="Invariant Violation: Could not find &quot;store&quot; in the context of &quot;Connect(PlayerForm)&quot;. Either wrap the root component in a &lt;Provider&gt;, or pass a custom React context provider to &lt;Provider&gt; and the corresponding React context consumer to Connect(PlayerForm) in connect options.
    at invariant (C:\ReactWS\2019_dev_005\node_modules\invariant\invariant.js:40:15)
    at Connect.renderWrappedComponent (C:\ReactWS\2019_dev_005\node_modules\react-redux\lib\components\connectAdvanced.js:162:32)
    at updateContextConsumer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:14944:19)
    at beginWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:15110:14)
    at performUnitOfWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17820:12)
    at workLoop (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17860:24)
    at renderRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:17946:7)
    at performWorkOnRoot (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18837:7)
    at performWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18749:7)
    at performSyncWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18723:3)
    at requestWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18592:5)
    at scheduleWork (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18401:5)
    at scheduleRootUpdate (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19069:3)
    at updateContainerAtExpirationTime (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19097:10)
    at updateContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19154:10)
    at ReactRoot.Object.&lt;anonymous&gt;.ReactRoot.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19416:3)
    at C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19556:14
    at unbatchedUpdates (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:18952:10)
    at legacyRenderSubtreeIntoContainer (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19552:5)
    at Object.render (C:\ReactWS\2019_dev_005\node_modules\react-dom\cjs\react-dom.development.js:19613:12)
    at Object.render (C:\ReactWS\2019_dev_005\node_modules\enzyme-adapter-react-16\build\ReactSixteenAdapter.js:352:114)
    at new ReactWrapper (C:\ReactWS\2019_dev_005\node_modules\enzyme\build\ReactWrapper.js:134:16)
    at Object.mount (C:\ReactWS\2019_dev_005\node_modules\enzyme\build\mount.js:21:10)
    at Object.mount (C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js:20:22)
    at Object.asyncJestLifecycle (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\jasmine_async.js:63:37)
    at resolve (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)" type="AssertionError">
      </failure>
      <failure message="Error: Cannot spy the handleNameChange property because it is not a function; undefined given instead
    at ModuleMockerClass.spyOn (C:\ReactWS\2019_dev_005\node_modules\jest-mock\build\index.js:697:15)
    at Object.spyOn (C:\ReactWS\2019_dev_005\src\test\PlayerForm.spec.js:30:22)
    at Object.asyncJestTest (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\jasmine_async.js:108:37)
    at resolve (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:56:12)
    at new Promise (&lt;anonymous&gt;)
    at mapper (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:43:19)
    at promise.then (C:\ReactWS\2019_dev_005\node_modules\jest-jasmine2\build\queue_runner.js:87:41)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)" type="AssertionError">
      </failure>
    </testcase>
    <system-out/>
    <system-err/>
  </testsuite>
</testsuites>